tests/Vagrantfile

```diff
@@ -0,0 +1,14 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure(2) do |config|
+  config.vm.box = "centos65"
+  config.vm.box_url = "http://downloads.juwai.io/devops/boxes/centos65-x86-64.box"
```

Comment:
It is just a simple CentOS65 box. That box url is now public I think.

On Mon, Apr 18, 2016 at 3:45 PM Alec Joseph Rivera <notifications@github.com<mailto:notifications@github.com>> wrote:

In tests/Vagrantfile<https://github.com/juwai/ansible-role-python27/pull/1#discussion_r60016487>:

> @@ -0,0 +1,14 @@
> +# -*- mode: ruby -*-
> +# vi: set ft=ruby :
> +
> +Vagrant.configure(2) do |config|
> +  config.vm.box = "centos65"
> +  config.vm.box_url = "http://downloads.juwai.io/devops/boxes/centos65-x86-64.box"


Do we have anything private on that box? It seems to be a pretty basic install of centos65.

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/juwai/ansible-role-python27/pull/1/files/25f18882895b19fdcfdb8f406a97b34566bb797d#r60016487>

'

[https://s3-ap-southeast-1.amazonaws.com/frontend-libraries/juwai/email_footer_images/Logo_com_logo.jpg]<http://www.juwai.com>

Hong Kong +852 2850 5083
香港湾仔湾仔道133号卓凌中心35楼
Level 35, Times Media Centre, 133 Wan Chai Road, Wan Chai, Hong Kong

Shanghai +86 21 6072 9628
宏居网络技术（上海）有限公司 上海虹口区东大名路687号501室
Suite 501, 1929 Complex, 687 Daming Road East, Hongkou District, Shanghai, China

Join us: list.juwai.com<http://list.juwai.com?utm_source=Outlook&utm_medium=Email&utm_campaign=SignatureV1> See us: www.juwai.com<http://www.juwai.com> Follow us: [Facebook] <https://www.facebook.com/juwailimited>  [Twitter] <https://twitter.com/juwailimited>  [Google+] <https://plus.google.com/+juwai/>  [Linkedin] <http://www.linkedin.com/company/juwai-com>  [Weibo] <http://weibo.com/juwaicom>

[https://list.juwai.com/uploads/media/blog/0001/02/d45eaa25a0c9947343e700621c1b47a5daa709c8.jpeg] <https://list.juwai.com/global-developer-summit-beijing-2016?utm_source=Emarsys&utm_medium=banner&utm_campaign=Golbal%20developer%20summit%20beijing%202016>

This e-mail message may contain confidential or legally privileged information and is intended only for the use of the intended recipient(s). Any unauthorized disclosure, dissemination, distribution, copying or the taking of any action in reliance on the information herein is prohibited.

'